redash/models/__init__.py

```diff
@@ -791,14 +791,25 @@ def evaluate(self):
         data = json_loads(self.query_rel.latest_query_data.data)
 
         if data['rows'] and self.options['column'] in data['rows'][0]:
+            operators = {
+                '>': lambda v, t: v > t,
+                '>=': lambda v, t: v >= t,
+                '<': lambda v, t: v < t,
+                '<=': lambda v, t: v <= t,
+                '==': lambda v, t: v == t,
+                '!=': lambda v, t: v != t,
+
+                # backward compatibility
+                'greater than': lambda v, t: v > t,
+                'less than': lambda v, t: v < t,
+                'equals': lambda v, t: v == t,
+            }
+            should_trigger = operators.get(self.options['op'], lambda: False)
```

Comment:
The fallback lambda should take arguments as well.

Reply:
Ooops... Done :+1: